content/operate/kubernetes/release-notes/7-22-0-releases/7-22-0-15-july2025.md

```diff
@@ -0,0 +1,81 @@
+---
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: Feature release with Helm chart general availability, Kubernetes 1.33 and OpenShift 4.19 support, Redis Software 7.22.0-216 support, and enhancements.
+hideListLinks: true
+linkTitle: 7.22.0-15 (July 2025)
+title: Redis Enterprise for Kubernetes 7.22.0-15 (July 2025) release notes
+weight: 1
+---
+
+## Highlights
+
+* Helm chart support is now generally available.
+
+## New in this release
+
+Redis Enterprise for Kubernetes 7.22.0-15 is a feature release that includes enhancements, platform updates, and support for Redis Software 7.22.0-216.
+
+### Enhancements
+
+* Call home - support HTTP proxy configuration (RED-156935)
+* Helm - optimize installation time (RED-153889)
+* Add support for migrating to a Helm-based installation from a non-Helm installation (RED-113099)
+* Support upgrade using Helm (RED-153890)
+* Support any custom port for database service (different from cluster database port) (RED-156946)
+* Kubernetes platform updates, support for Kubernetes 1.33
+* Support for Redis Software 7.22.0-216 (RED-163025)
+
+### Resolved Issues
+
+* Adding `servicesAnnotations` in the REC spec keeps updating the services in a loop (RED-154777, RED-158816)
+* New call home job is not created if the old one is stuck due to pod in `imagePullBackoff` (RED-157526)
+* REAADB ACL roles do not get properly assigned according to `rolesPermissions` in spec (RED-158955)
+* CVE fixes
+* `allowPrivilegeEscalation` is unset in the init container for `readOnlyRootFilesystem` (RED-163637)
```

Comment:
Should we add a more general note in the "enhancements" section?
> Hardened the securityContext of the init containers created when readOnlyRootFilesystem is enabled.

The change we did included setting a value for `allowPrivilegeEscalation`, as well as some additional attributes in the security context of these containers, so we could mention that.